src/components/compound/DlSlider/components/DlSliderInput.vue

```diff
@@ -66,9 +75,24 @@ export default defineComponent({
             if (stateManager.disableDebounce) {
                 return handleChange
             }
-            return debounce(handleChange, 100)
+            // return debounce(handleChange, 100)
```

Comment:
I removed the debounce to prevent non-numeral characters from showing for a moment, since I changed the input type to `text`.

And I will remove the comment.